client/test/config.json

```diff
@@ -113,6 +113,6 @@
 		"users": {
 			"test": [ "actions", "reducer", "selectors" ]
 		},
-		"test": [ "index" ]
+		"test": [ "index", "initial-state" ]
```

Comment:
Yes, we missed that one ...
I figured out yesterday: https://github.com/Automattic/wp-calypso/commit/4ed280bf9e706a56a00fe0b36258e44dafda7eda, but it isn't merged yet :)